### DISCUSSION TITLE
Does keypoints dataset horizontal flip augmentation automatically enabled? How to disable it?

### DISCUSSION THREAD

**Discussion by lucasjinreal** [facebookresearch/detectron2#4794] [Q&A]
Hi, i want disable keypoints flip augmentation without change d2 source code, can't find a way to enable or disable it, any docs?

**Comment by ppwwyyxx** [comment]
yacs config: `cfg.INPUT.RANDOM_FLIP="none"`
lazy config: edit the list of augmentations in config directly
